client/blocks/reader-post-card/index.jsx

```diff
@@ -27,6 +27,79 @@ import { getDiscoverBlogName,
 } from 'reader/discover/helper';
 import DiscoverFollowButton from 'reader/discover/follow-button';
 
+function trackable( TrackedComponent ) {
```

Comment:
Definitely lets move forward with this! 
We can separate it out into its own component